.github/workflows/on-push.yml

```diff
@@ -283,6 +283,8 @@ jobs:
           source $(poetry env info --path)/bin/activate
           cd content
           rm -rf .pre-commit-config_template.yaml
+          rm -f Tests/docker_native_image_config.json
+          touch .private-repo-settings
```

Comment:
why is this file needed?

Reply:
Because I thought to use the `is_external_repository` function and it checks against this file.